sashimi/state.py

```diff
@@ -472,12 +480,21 @@ def n_planes(self):
 
     @property
     def save_params(self):
-        return convert_save_params(
-            self.save_settings,
-            self.volume_setting,
-            self.camera_settings,
-            self.trigger_settings,
-        )
+        if self.global_state == GlobalState.PLANAR_PREVIEW:
+            save_p = convert_save_params(
+                self.save_settings,
+                self.single_plane_settings,
+                self.camera_settings,
+                self.trigger_settings,
+            )
+        else:
+            save_p = convert_save_params(
+                self.save_settings,
+                self.volume_setting,
+                self.camera_settings,
+                self.trigger_settings,
+            )
+        return save_p
```

Comment:
inside `save_params` checks if we are in planar mode and returns the planar settings - if not defaults to volume settings